modules/standard/Regexp.chpl

```diff
@@ -18,12 +18,302 @@
  */
 
 /*
-   Regexp Module
+
+This module provides regular expression support.
+
+The regular expression support is built on top of the RE2 regular expression
+library. As such, the exact regular expression syntax available is the syntax
+from RE2, which is available at http://code.google.com/p/re2/wiki/Syntax and
+included here for your convenience.
+
+
+Enabling Regular Expression Support
+-----------------------------------
+
+Setting the environment variable CHPL_REGEXP to re2 will enable regular
+expression support with the RE2 library:
+
+.. code-block:: sh
+
+   export CHPL_REGEXP=re2
+
+Then, rebuild Chapel. As part of the make, the RE2 library will be
+downloaded from its Mercurial repository at re2.googlecode.com.
```

Comment:
I think re2 has moved to github --> https://github.com/google/re2

As a side note, sphinx will make links automatically out of things that start with http:// or https://.